### PR TITLE
Blog list page java script#005

### PR DESCRIPTION
**Pull request by Salim-54**
#### What does this PR do?

- interact with button tabs
- javaScript hovering with DOM propagation

#### Description of Task to be completed? 

As a user, I should be able
- To interact with button tabs
- Read more details  for every article 
- To see the Navigation bar which makes visible hovered link and makes invisible the rest.
#### How should this be manually tested?

- Clone the repo

- checkout to branch ``blogListPage-JavaScript#005``

#### Any background context you want to provide?

N/A

#### What are the relevant Pivotal Tracker/trello stories?

- [link to my pivotal tracker/tello/jira ... implemented story](https://trello.com/c/ewynI2Db/22-bloglistpage-javascript005)

#### Screenshots (if appropriate)

Screenshot (254)
Screenshot (251)
Screenshot (252)
Screenshot (253)

#### Questions:

- N/A

### CODE REVIEW

File: css/articles.css [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  padding: 0;
+  box-sizing: inherit; }
+
+  *::selection {
+    color: black;
+    background: #F4F5F5; }
+
+  *::after, *::before {
+    box-sizing: inherit; }
+
```
Comment (PrinceNiyonshuti) [inline css/articles.css:12]: kindly , indent well the codes like 
`*::after, *::before {
    box-sizing: inherit; 
}`

File: views/articles.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Baloo Paaji 2">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Heebo">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Arial">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sansation Light">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Changa">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+
+
```
Comment (PrinceNiyonshuti) [inline views/articles.html:19]: Try to remove the space gap , otherwise nice work

**Review by PrinceNiyonshuti — CHANGES_REQUESTED** [review]
**Working on the indentation of the code**
- try to indent codes where necessary mostly in the CSS file
- on the article HTML file try to remove the empty spaces

**Review by Xldivin — COMMENTED** [review]
clean codess

**Pull request merged**